### PR TITLE
fix(autosuggest): add default min-width to autosuggest when loading

### PR DESCRIPTION
**Pull request by Nantawat-Poothong**
## Description

In case autosuggest asynchronous with mask, If we didn't set default min-width for autosuggest, the result when opening autosuggest is the mask didn't show, because autosuggest width doesn't set.

- Add css min-width to autosuggest when loading
- Fix backdrop opacity

Fixes # (issue)
https://jira.refinitiv.com/browse/ELF-1845

### CODE REVIEW

File: packages/elemental-theme/src/custom-elements/ef-autosuggest.less [hunk 1]
```diff
@@ -52,7 +53,7 @@
 
   [part=backdrop] {
     background-color: @overlay-background-color;
-    opacity: @overlay-opacity;
```
Comment (goremikins) [inline packages/elemental-theme/src/custom-elements/ef-autosuggest.less]: Why did you override this?

Comment (dtanp-rft) [reply packages/elemental-theme/src/custom-elements/ef-autosuggest.less]: The style looks wrong to me. Not sure if reuse overlay-opacity here is a right choice. I mean we may keep the same opacity but change background color instead if it's better solution. 

Screen Shot 2565-06-06 at 17 32 12

Comment (goremikins) [reply packages/elemental-theme/src/custom-elements/ef-autosuggest.less]: It is wrong all together: being the loader too large and the background too grey. Perhaps we need a proper solution from Design/UX on how to cover this case (and all other cases with asynchronous popup loading). This code was just migrated from v3 where it made more sense.

Comment (goremikins) [reply packages/elemental-theme/src/custom-elements/ef-autosuggest.less]: maybe UX can say that entire screen needs to be greyed out, so we can deprecated autosuggest loading :)

Comment (dtanp-rft) [reply packages/elemental-theme/src/custom-elements/ef-autosuggest.less]: I'll go with this small improvement. It's better than today anyway. You know we would need to wait very long until they take an action. It means that users won't see any improvement until then :)

Comment (Nantawat-Poothong) [reply packages/elemental-theme/src/custom-elements/ef-autosuggest.less]: Updated!, I have revert override backdrop opacity :)

**Review by Sarin-Udompanish — APPROVED** [review]

**Pull request merged**